### DISCUSSION TITLE
The possibility of using MLIR to program on V70 card

### DISCUSSION THREAD

**Discussion by ngdymx** [Xilinx/mlir-aie#1409] [Q&A] [answered]
Hi all,

I am wondering if it supports V70 card? Thanks a lot

**Comment by jgmelber** [comment, marked as answer]
Currently our tool flow does not support V70. We will update here if anything changes.

**Reply by ngdymx** [reply]
Sure， thanks a lot

**Reply by ngdymx** [reply]
Hi,

 What is the tool that can be used to develop V70 now? Thanks a lot!

**Reply by jgmelber** [reply]
Currently it can only be used with Vitis AI
